pombola/templates/default_base.html

```diff
@@ -136,13 +138,20 @@
         {% endblock %}
 
         {% block js_end_of_body %}
-          <script src="//ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js"></script>
-          <script src="//www.google.com/jsapi"></script>
-          {% if settings.GOOGLE_RECAPTCHA_SITE_KEY %}
-              <script src="https://www.google.com/recaptcha/api.js" async defer></script>
-          {% endif %}
-          {% javascript 'base' %}
-          {% javascript 'survey' %}
+
+
+        <script src="//ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js"></script>
```

Comment:
if you rely on jquery 3.2.1 you'll have to remove this version and check that the site still works properly

Reply:
A lot of errors happen when I remove the previous Jquery

![Screenshot from 2021-11-03 18-09-53](https://user-images.githubusercontent.com/6994982/140088821-c877deb0-090c-45d2-a73b-d84c1de8b9e9.png)